erc/ERCS/erc-7562.md

```diff
@@ -249,44 +340,48 @@ That is, if the `validateUserOp()` is rejected for any reason, it is treated as
 
 * Definitions:
     * **`opsSeen`, `opsIncluded`, and reputation calculation** are defined above.
-    * `UnstakedReputation` of an entity is a maximum number of entries using this entity allowed in the mempool.
-    * `opsAllowed` is a reputation based calculation for an unstaked entity, representing how many `UserOperations` it is allowed to have in the mempool.
-* **[UREP-010]** `UserOperation` with unstaked sender are only allowed up to `SAME_SENDER_MEMPOOL_COUNT` times in the mempool.
+    * `UnstakedReputation` of an entity is the maximum number of entries using this entity allowed in the mempool.
+    * `opsAllowed` is a reputation-based calculation for an unstaked entity, representing how many `UserOperations` it is allowed to have in the mempool.
+* **[UREP-010]** `UserOperation` with unstaked sender is only allowed up to `SAME_SENDER_MEMPOOL_COUNT` times in the mempool.
 * **[UREP-020]** For other entities: \
-    `opsAllowed = SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT + (inclusionRate * INCLUSION_RATE_FACTOR) + (min(opsIncluded, 10000)`.
-    * This is a default of `SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT` for new entity
+  `opsAllowed = SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT + (inclusionRate * INCLUSION_RATE_FACTOR) + (min(opsIncluded, 10000)`.
```

Comment:
I don't understand this calculation.  @shahafn did you mean to have `+` in the 2nd part of the expression?  The parentheses are also mismatched (the last one is not closed). 

As it is now, the expression sums three values:
- SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT - always 10
- (inclusionRate * INCLUSION_RATE_FACTOR) - with inclusionRate being opsIncluded/opsSeen (max 1) and INCLUSION_RATE_FACTOR of 10, this value is at most 10.
- (min(opsIncluded, 10000) - climbs up to 10000 and stays there forever because it is not affected by opsSeen.

So opsAllowed is basically `10 + 10 + opsIncluded`, and doesn't drop when the entity starts misbehaving.  Please check the calculation.

Reply:
Yes, I think it should have been 
`opsAllowed = SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT + inclusionRate * (INCLUSION_RATE_FACTOR + (min(opsIncluded, 10000)`